src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-weight: 300px;
+  margin-top: 12px;
+}
+
+.search-field {
+  margin-bottom: 20px;
+}
+
+.big-form {
+  width: 100%;
+  background: url("./images/Search.svg")  left  no-repeat;
+  background-position: 26px 50%;
+  background-size: 19px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  color: #3d4e61;
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.big-form:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big-form:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
+
+.small-form {
```

Comment:
you repeat lot of properties in big and small forms selectors
create a common selector